src/objects/LOD.js

```diff
@@ -74,6 +74,23 @@ class LOD extends Object3D {
 
 	}
 
+	removeLevel( distance ) {
+
+		const levels = this.levels;
+		levels.map( ( _level, index ) => {
+
+			if ( _level.distance === distance ) {
+
+				levels.splice( index, 1 );
```

Comment:
I'm not using `map()` that often so: Is it safe to remove an array instance inside the callback while processing the array? You couldn't do that with a normal for loop with ascending index.

Reply:
Yep there's no problem. I'm using the map just to loop over the levels.

Now if you prefer I can change that for a "for loop", depending on you, but in the end it is the same.

PS: I tested it in the current LOD example, but I didn't want to change it

Reply:
I think `const levels = this.levels.filter(_level => _level.distance !== distance)` maybe better.

Reply:
We mostly use `for` loop in the code base so for consistency reasons I vote for a reverse for loop.

Reply:
Done :)